README.md

```diff
@@ -125,6 +124,16 @@ In addition to this, set the `aria-current="page"` attribute on the relevant `<a
 
 This is to indicate the user’s position in the navigation if they can’t perceive the visual highlight styling.
 
+## Point the 'Sign out' link at your service's sign out endpoint
```

Comment:
The other bullet points here are H3s - do you want this one to be the same?

Reply:
I certainly do - good spot!